pkg/service/v1alpha1/controller.go

```diff
@@ -162,8 +162,20 @@ func (cs *controller) ControllerExpandVolume(
 	ctx context.Context,
 	req *csi.ControllerExpandVolumeRequest,
 ) (*csi.ControllerExpandVolumeResponse, error) {
+	updatedSize := req.GetCapacityRange().GetRequiredBytes()
+	if err := utils.ResizeVolume(req.VolumeId, updatedSize); err != nil {
+		return nil, status.Errorf(
+			codes.Internal,
+			"failed to handle ControllerExpandVolumeRequest for %s, {%s}",
+			req.VolumeId,
+			err.Error(),
+		)
+	}
+	return csipayload.NewControllerExpandVolumeResponseBuilder().
+		WithCapacityBytes(updatedSize).
+		WithNodeExpansionRequired(true).
+		Build(), nil
```

Comment:
We can have add some status checks b/w `CVC.Status.Capacity` and `CVC.Spec.Capacity` here before returning `ControllerExpandVolumeResponse` to

Reply:
some thing like 
```go
        actualSize := cvc.Status.Capacity[corev1.ResourceStorage]
	desiredSize := cvc.Spec.Capacity[corev1.ResourceStorage]

	if actualSize.Cmp(desiredSize) != 0 {
		return nil, status.Error(codes.Internal, fmt.Sprintf("ControllerExpandVolume failed to resize: %v", err))
	}

        return csipayload.NewControllerExpandVolumeResponseBuilder().
		WithCapacityBytes(updatedSize).
		WithNodeExpansionRequired(true).
		Build(), nil
```